Dockerfile

```diff
@@ -1,10 +1,9 @@
-# impish is needed since glibc 2.34 is required for linux 5.15.30
-FROM ubuntu:impish-20220316
+FROM ubuntu:jammy-20220815
```

Comment:
What do you think of using debian instead? (probably more stable in the long run).

```suggestion
# bookworm is needed since glibc 2.34 is required for linux 5.15.30
FROM debian:bookworm-slim
```

(the README should be updated as well)

Reply:
Fair enough. Could you keep the comment in some form?

`glibc 2.34 is required for linux 5.15.30 (available since ubuntu:impish or debian:bookworm)`

Reply:
Sure, added the comment.